CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 
 * [#2467](https://github.com/ruby-grape/grape/pull/2467): Fix repo coverage - [@ericproulx](https://github.com/ericproulx).
 * [#2468](https://github.com/ruby-grape/grape/pull/2468): Align `error!` method signatures across different places - [@numbata](https://github.com/numbata).
+* [#2469](https://github.com/ruby-grape/grape/pull/2469): Fixes #2347 - Correct full path building for lateral scopes - [@numbata](https://github.com/numbata).
```

Comment:
Let’s say “Fix” to match other lines.

Reply:
Done! ([diff](https://github.com/ruby-grape/grape/compare/58d54c34e2c33473063eca40cb9bbd3ebb0b4472..cc54313ef17063d19453b5c8ee052536b83f116c#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edL11))